CHANGES_NEXT_RELEASE

```diff
@@ -1,3 +1,4 @@
+[cygnus-ngsi, cygnus-common][MySQLSQLSink] Use of the upsert transaction (#1806, #1936)
 [cygnus-ngsi, cygnus-common][PosgtgreSQLSink, PostgisSink] Create upsert transaction (#1806, #1936)
```

Comment:
Maybe both lines could be joined:

```suggestion
[cygnus-ngsi, cygnus-common][PosgtgreSQLSink, PostgisSink, MySQLSQLSink] Create upsert transaction (#1806, #1936)
```

Reply:
I agreee.

Fixed in [803356b](https://github.com/telefonicaid/fiware-cygnus/pull/1944/commits/803356bc6288c2273ffb39529151d556fb54ba58)